### PR TITLE
Change log for v49.7.0

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Change log for v49.7.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+#### salesforcedx-vscode-lwc
+
+- Update to the latest version of the LWC language server [PR #2380](https://github.com/forcedotcom/salesforcedx-vscode/pull/2380)
+
+#### docs
+
+- Fix example under [Advanced Setup](https://developer.salesforce.com/tools/vscode/en/getting-started/java-setup/#advanced-setup) on Java Setup page [PR #2426](https://github.com/forcedotcom/salesforcedx-vscode/pull/2426) - Contribution by ([@nabondance](https://github.com/nabondance))
+
 # 49.5.0 - August 13, 2020
```
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md:17]: Did 49.6.0 get left out?

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:17]: Yes, I think we forgot to port that change back to `develop`

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+# 49.7.0 - August 25, 2020
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:1]:
```suggestion
# 49.7.0 - August 27, 2020
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+
+## Fixed
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:3]: These two can be merged into one item

``` 
#### salesforcedx-vscode-lightning, salesforcedx-vscode-lwc

- Update ...
```

**Comment by brpowell** [comment]
@ekapner change log is ready when you have a free moment!

**Pull request merged**